truss-chains/truss_chains/remote.py

```diff
@@ -124,7 +127,7 @@ def _push_service(
         )
     elif isinstance(options, definitions.PushOptionsBaseten):
         with utils.log_level(logging.INFO):
-            service = _push_to_baseten(truss_dir, options)
+            service = _push_to_baseten(truss_dir, options, chainlet_descriptor.name)
```

Comment:
do we want the `name` or the `display_name` here?

Reply:
looks like we use `display_name` when we send the chainlet mapping over 
https://github.com/basetenlabs/truss/blob/main/truss-chains/truss_chains/remote.py#L316-L320

Reply:
great catch ty